bower.json

```diff
@@ -27,6 +27,7 @@
     "ember": "2.2.0",
     "jquery": "1.9.1 - 2",
     "moment": ">=2.9.0",
-    "bootstrap": ">= 2.3.2"
+    "bootstrap": ">= 2.3.2",
+    "fabric": "~1.4.*"
```

Comment:
Since darkroom was removed, we can probably remove the fabric resolution as well. (and avoid one unnecessary alert message on the console)